__tests__/pet.test.js

```diff
@@ -0,0 +1,135 @@
+const Pet = require('../src/pet');
+
+describe('Pet', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('isAlive', function () {
+    it('check if pet is dead', () => {
+        const max = new Pet('max');
+        max.hunger = 10;
+        max.fitness = 0;
+        max.age = 35;
+        max.isAlive;
+        expect(max.isAlive).toBe(false);
+    })
```

Comment:
I would say you could break this up into several tests, as there are multiple possible ways that could result in the pet dying, eg
```
  it("check if pet dies of old age", () => {
    const max = new Pet("max");
    max.age = 35;

    expect(max.isAlive).toBe(false);
  });

  it("check if pet dies of hunger", () => {
    const max = new Pet("max");
    max.hunger = 10;

    expect(max.isAlive).toBe(false);
  });

  it("check if pet dies of low fitness", () => {
    const max = new Pet("max");
    max.fitness = 0;

    expect(max.isAlive).toBe(false);
  });
```
Note you dont have to call `isAlive` separately as you are on line 16, you can just do it in the assertion at the end